src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -190,6 +209,70 @@ def get_rate(cls, svc_type, svc_name, path):
             return differentiate(*data[-2:])
         return 0.0
 
+    @classmethod
+    def get_client_perf(cls):
```

Comment:
Maybe I'm wrong, but it looks like API tests are missing for the new methods here, if so, make sure to add them to avoid any regression.

Reply:
Ack.

Reply:
API tests are crucial here in order to find out regressions early on, which could be introduced by anyone working on Ceph.

Reply:
Right, and as per our previous conversation, I've created this test coverage issue:
https://tracker.ceph.com/issues/27220

Reply:
As per our standup conversation, please add these tests in order to maintain test coverage before this PR gets merged. Thanks!